2.4/root/usr/bin/run-mongod

```diff
@@ -114,8 +103,7 @@ else
   # `cleanup` references it.
   (
     unset_env_vars
-    mongod $mongo_common_args --replSet ${MONGODB_REPLICA_NAME} \
-      --keyFile ${MONGODB_KEYFILE_PATH} ${auth_args}
+    mongod $mongo_common_args --replSet ${MONGODB_REPLICA_NAME} ${auth_args}
```

Comment:
I agree with quoting  `MONGODB_REPLICA_NAME`. I don't know why `auth_args` are separated (Also this is not necessary for MongoDB 2.6+ because `--keyFile` implies auth). So `--auth` could be simply added to `mongo_common_args` for MongoDB 2.4.

@rhcarvalho Are you fine with doing it in this PR?